core/src/main/java/org/testcontainers/utility/CommandLine.java

```diff
@@ -23,6 +23,8 @@
 
     private static final Logger LOGGER = getLogger(CommandLine.class);
 
+    private static String OS = System.getProperty("os.name").toLowerCase();
```

Comment:
I _think_ we can make this `final`, which should be better from a naming perspective (plus it should probably never change!)